src/sentry_plugins/vsts/client.py

```diff
@@ -18,7 +18,7 @@ class VstsClient(AuthApiClient):
 
     def request(self, method, path, data=None, params=None):
         headers = {
-            "Accept": "application/json; api-version={}".format(self.api_version),
+            u"Accept": "application/json; api-version={}".format(self.api_version),
```

Comment:
```suggestion
            u"Accept": u"application/json; api-version={}".format(self.api_version),
```

Reply:
@joshuarli hahah oops, I put the unicode on the wrong string

Reply:
Are you making these changes manually? I'd search for any `format` strings and make sure both the format string and the arguments are unicode. See also https://github.com/getsentry/sentry/pull/21872#discussion_r520068775

Reply:
@joshuarli I was doing changes manually but searching for `format` within `plugins`.